sre-recipes/sandboxctl

```diff
@@ -161,6 +172,23 @@ def create(project, *args, **kwargs):
         if not complete:
             os.killpg(0, signal.SIGTERM)
 
+@cli.command()
+def describe():
+    """Show information about an existing sandbox"""
+    gcp_path = "https://console.cloud.google.com"
+    loadgen_ip = get_loadgen_ip()
+    external_ip = get_external_ip()
+    project_id = get_project_id()
+    gcp_kubernetes_path = gcp_path + '/kubernetes/workload?project=' + project_id
+    gcp_monitoring_path = gcp_path + '/monitoring?project=' + project_id
+    print("""
+    Cloud Operations Sandbox info for project {}:
```

Comment:
nit: it would be best if the order of these fields matched the [initial success message](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/a46746143530cdad0c371931ccb88e953a0afa40/terraform/install.sh#L328) (I wonder if we should even change that block to just call `sandboxctl describe`?)

Reply:
Moved project_id, but not sure what you meant, can you clarify the brackets?

Reply:
To match the [initial success message](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/a46746143530cdad0c371931ccb88e953a0afa40/terraform/install.sh#L328), it should look like this:

```
Cloud Operations Sandbox info for project {}:
    - Google Cloud Console KBE Dashboard: {}
    - Google Cloud Console Monitoring Workspace {}
    - Load generator web interface: {}
    - Hipstershop web app address: {}""".format(project_id, loadgen_ip, external_ip, gcp_kubernetes_path, gcp_monitoring_path)
```

In the brackets, I was suggesting maybe we should call this function to print the success message instead of having similar code in two places. But probably not worth changing at this point